src/section-exponent-rules-and-scientific-notation.ptx

```diff
@@ -311,52 +161,68 @@
-                    <mrow>  \amp= 3^{5}x^{6} </mrow>
+                <p>We can simplfy this using the <xref ref="exponent-rules-quotient" autoname="title" />.  We'll apply this rule to both bases, <m>3</m> and <m>x</m>:
+                <md>
+                    <mrow>\frac{3^7x^9}{3^2x^3} \amp= 3^{7-2}\cdot x^{9-3} </mrow>
+                    <mrow>\amp= 3^{5}x^{6} </mrow>
+                    <mrow>\amp= 243x^6</mrow>
                 </md></p>
             </solution>
-        </example>-->
+        </example>
         
         <paragraphs>
             <title>Quotient to a Power Rule</title>
-            <p>We have the <xref ref="exponent-rules-product-to-a-power" autoname="title" />, which applies the outer exponent to each factor in the product inside the parentheses.  What happens if we apply an exponent to a quotient instead?</p>
+            <p>We have the <xref ref="item-exponent-rules-product-to-a-power">Property</xref>, which applies the outer exponent to each factor in the product inside the parentheses.  We can use the rules of fractions to extend this property to a quotient raised to a power.</p>
         </paragraphs>
 
         <example>
             <statement>
-                <p>Let <m>y</m> be a real number, where <m>y \neq 0</m>. Simplify <m>\left( \dfrac{7}{y} \right)^4</m>.</p>
+                <p>Let <m>y</m> be a real number, where <m>y \neq 0</m>. Simplify <m>\left(\dfrac{7}{y}\right)^4</m>.</p>
             </statement>
             <solution>
                 <p>Writing the expression without an exponent and then simplifying, we have:<md>
                     <mrow>\left( \frac{7}{y} \right)^4 \amp= \left( \frac{7}{y} \right) \left( \frac{7}{y} \right) \left( \frac{7}{y} \right) \left( \frac{7}{y} \right)</mrow>
                     <mrow>\amp= \frac{7 \cdot 7 \cdot 7 \cdot 7}{y \cdot y \cdot y \cdot y}</mrow>
                     <mrow>\amp= \frac{7^4}{y^4}</mrow>
+                    <mrow>=amp= \frac{2401}{y^4}</mrow>
                 </md>Similar to the product to a power rule, we essentially applied the outer exponent to the entire numerator and denominator. </p>
             </solution>
         </example>
 
-        <p>The previous example shows how the <term>Quotient to a Power Rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator.</p>
+        <p>The previous example shows how the <term>quotient to a power rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me></p>
+        <p>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator. This means that we can directly compute <m>\left(\dfrac{7}{y}\right)^4</m>:
```

Comment:
Thinking out loud, not necessarily making a suggestion. Do we want to mix vocabulary terms from {"fraction", "numerator", "denominator"} with vocabulary terms from {"quotient", "dividend", "divisor"}?